### ISSUE TITLE
Remove Animations with "Safe Display" Enabled 

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#7163]
- [ ] The only animations that I still see are the active scrolling when scrolling the deck list, which I do often enough that it is annoying, and the sync spinner, which I do infrequently enough that it is not a real issue.
- [ ] If at all possible, when animations are disabled, a swipe up or down on the deck list should probably move the list a proportionate amount without actually showing the scrolling animation.

_Originally posted by @dotancohen in https://github.com/ankidroid/Anki-Android/issues/7114#issuecomment-695342582_

**Comment by david-allison-1** [comment]
I did have an attempt at removing the syncing progress bar a couple days ago when it first came up and it was more complex than I'd expected due to the way the progress spinner was hardcoded.

**Comment by dotancohen** [comment]
still searching for solutions

**Comment by dotancohen** [comment]
still searching for solutions

**Comment by dotancohen** [comment]
still looking for solutions

**Comment by mikehardy** [comment]
One of these days :-)

**Comment by dae** [comment]
@dotancohen are you still using that device? Sadly this doesn't look like something we'll find time for soon.

**Comment by dotancohen** [comment]
> @dotancohen are you still using that device? Sadly this doesn't look like something we'll find time for soon.

Yes, we have two of them in use, both are of this configuration:
```
shell@ntx_6sl:/ $ getprop ro.build.version.release
4.4.2

shell@ntx_6sl:/ $ getprop ro.build.version.sdk
19

shell@ntx_6sl:/ $ getprop ro.product.cpu.abi
armeabi-v7a

shell@ntx_6sl:/ $ cat /proc/cpuinfo
Processor       : ARMv7 Processor rev 10 (v7l)
BogoMIPS        : 790.52
Features        : swp half thumb fastmult vfp edsp neon vfpv3n
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x2
CPU part        : 0xc09
CPU revision    : 10

Hardware        : Freescale i.MX 6SoloLite NTX Board
Revision        : 60200
Serial          : 162709d4ea9ce69b
```

I really don't mind if the current bugs are not fixed, as the most recent AnkiDroid version that device can run is v2.13.5. So long as that version can still sync with AnkiWeb, we'll be fine. Thank you!

**Comment by Sanskriti0805** [comment]
hello is this issue still open ?

**Comment by SanjaySargam** [comment]
@Sanskriti0805 I'd suggest you to start with good first issues first.

**Comment by dotancohen** [comment]
I'm still using those devices. But this issue is applicable to all E-Ink devices, not just those old ones. I know of multiple people using AnkiDroid on Boox devices, one of which is in colour.

In fact, if you guys need screenshots or video of E-Ink screens, B&W or colour, I'm glad to help.

**Comment by mikehardy** [comment]
I think everyone likes the idea of preserving the functionality, I believe we just need PRs that remove the animations
